openpype/pipeline/stagingdir.py

```diff
@@ -0,0 +1,212 @@
+import os
+import tempfile
+from openpype.lib import (
+    Logger,
+    filter_profiles
+)
+from openpype import pipeline
+from openpype.settings import (
+    get_project_settings
+)
+
```

Comment:
Use relative imports (would require to modify usages in code).
```suggestion
from openpype.settings import get_project_settings

from .anatomy import Anatomy
from .tempdir import create_custom_tempdir
```